sway-lib-std/src/i8.sw

```diff
@@ -0,0 +1,141 @@
+library i8;
+
+use core::num::*;
+use ::assert::assert;
+
+/// The 8-bit signed integer type.
+/// Represented as an underlying u8 value.
+pub struct i8 {
+    underlying: u8,
+}
+
+pub trait From {
+    /// Function for creating i8 from its u8 and bool components.
+    fn from(value: u8) -> Self;
+}
+
+impl From for i8 {
+    fn from(value: u8) -> i8 {
+        i8 {
+            underlying: value,
+        }
+    }
+}
+
+impl core::ops::Eq for i8 {
+    pub fn eq(self, other: i8) -> bool {
+        self.underlying == other.underlying
+    }
+}
+
+impl core::ops::Ord for i8 {
+    pub fn gt(self, other: Self) -> bool {
+        self.underlying > other.underlying
+    }
+
+    pub fn lt(self, other: Self) -> bool {
+        self.underlying < other.underlying
+    }
+}
+
+impl i8 {
+    pub fn indent() -> u8 {
+        128u8
+    }
+}
+
+impl i8 {
+    /// Initializes a new, zeroed i8.
+    pub fn new() -> i8 {
+        i8 {
+            underlying: ~i8::indent(),
+        }
+    }
+
+    /// The smallest value that can be represented by this integer type.
+    pub fn min() -> i8 {
+        i8 {
+            underlying: ~u8::min(),
+        }
+    }
+
+    /// The largest value that can be represented by this type,
+    pub fn max() -> i8 {
+        i8 {
+            underlying: ~u8::max(),
+        }
+    }
+
+    /// The size of this type in bits.
```

Comment:
Similar to the `U256` docs, can you include the actual max, min, and number of bits in the docstrings?